tfe/resource_tfe_variable_set_test.go

```diff
@@ -11,67 +11,10 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
-func TestAccTFEVariableSet_basic(t *testing.T) {
```

Comment:
Why are all these tests removed?

Reply:
Most of the tests here were specifically for using `workspace_ids` as an argument so I removed them. I can add another test step which uses a config that creates a `tfe_workspace_variable_set` and checks `workspace_ids`.